### PR TITLE
Add docker instructions

### LINKED ISSUE #190: docs: add local dev with docker instructions to the contributing guide.
Import this guide (https://github.com/JayjeetAtGithub/skyhook-nsdi/tree/master/docker_build) into the contributing guide in this repository.

### PR DESCRIPTION
**Pull request by YashJipkate**
Fix #190

### CODE REVIEW

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md [hunk 1]
```diff
@@ -76,3 +76,31 @@ docker-compose run --service-ports ubuntu-cls-demo
...
+```
+
+3. Install docker.
+```bash
+curl -o- https://get.docker.com | bash
+```
+
+3. Run the build script.
+```bash
+./build.sh
```
Comment (JayjeetAtGithub) [inline cpp/src/arrow/adapters/arrow-rados-cls/README.md:100]: Where is this script ?

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md [hunk 2]
```diff
@@ -76,3 +76,31 @@ docker-compose run --service-ports ubuntu-cls-demo
...
+```
+
+4. Run the tests script.
+```bash
+./test.sh
```
Comment (JayjeetAtGithub) [inline cpp/src/arrow/adapters/arrow-rados-cls/README.md:105]: Same for this

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md [hunk 3]
```diff
@@ -76,3 +76,31 @@ docker-compose run --service-ports ubuntu-cls-demo
 # Development
 
 Check out these [instructions](docs/contributing.md) for setting up a local development environment.
+
+## Set up the development environment using docker
```
Comment (JayjeetAtGithub) [inline cpp/src/arrow/adapters/arrow-rados-cls/README.md:80]: Can you add this guide in the contributing.md instead of here ?

**Review by JayjeetAtGithub — CHANGES_REQUESTED** [review]

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md (hunk 2 above)
Comment (YashJipkate) [reply cpp/src/arrow/adapters/arrow-rados-cls/README.md:105]: This was in the README instructions that was mentioned in the issue. Is there some other way I should import them as was said in the issue?

Comment (JayjeetAtGithub) [reply cpp/src/arrow/adapters/arrow-rados-cls/README.md:105]: Yes, bring the `build.sh` and `test.sh` scripts to this repo also.

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md (hunk 3 above)
Comment (YashJipkate) [reply cpp/src/arrow/adapters/arrow-rados-cls/README.md]: Done

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md (hunk 2 above)
Comment (YashJipkate) [reply cpp/src/arrow/adapters/arrow-rados-cls/README.md]: Okay. Done.

File: cpp/src/arrow/adapters/arrow-rados-cls/README.md (hunk 1 above)
Comment (YashJipkate) [reply cpp/src/arrow/adapters/arrow-rados-cls/README.md]: Brought the scripts. Done.

File: cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md [hunk 4]
```diff
@@ -17,7 +17,37 @@
...
+```
+
+3. Install docker.
+```bash
+curl -o- https://get.docker.com | bash
+```
+
+3. Run the build script.
+```bash
+../docker/build.sh
```
Comment (JayjeetAtGithub) [inline cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md:42]: Let's have all the scripts under the `arrow-rados-cls/scripts` directory.

File: cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md [hunk 5]
```diff
@@ -17,7 +17,37 @@
...
+```
+
+4. Run the tests script.
+```bash
+../docker/test.sh
```
Comment (JayjeetAtGithub) [inline cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md:47]: Same here.

**Review by JayjeetAtGithub — CHANGES_REQUESTED** [review]
Thanks a lot @YashJipkate . Just a minor change required.

File: cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md (hunk 5 above)
Comment (YashJipkate) [reply cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md]: Done.

**Comment by JayjeetAtGithub** [comment]
Thanks @YashJipkate

**Pull request merged**